### ISSUE TITLE
Succinate dehydrogenase positive muscle fibers

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6373]
**Preferred term label:**
Succinate dehydrogenase positive muscle fibers

**Synonyms**
SDH positive muscle fibers

**Definition (free text, please give PubMed ID)**
The histochemical assay for SDH is used to distinguish between oxidative and nonoxidative (actually, “less” oxidative) fibers. Oxidative fibers have a relatively dense, purple speckled appearance, while nonoxidative fibers have only scattered purple speckles. Therefore, this histochemical assay reflects the relative oxidative potential of muscle fibers.

**Parent term (use hpo.jax.org/app)**
Muscle abnormality related to mitochondrial dysfunction HP:0003800

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mitochondrial disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MLK6014  -- Terms in the HPO always describe Abnormalities. This feature is not in itself abnormal. An abnormal feature would be something like "Reduced SDH activity in ....". I am going to close this item with the request to make new issues for "Abnormality" features as required. 
Thanks!
